MIGRATION.md

```diff
@@ -571,6 +704,73 @@ Starting in 8.0, Storybook requires Angular 15 and up.
 
 Starting in 8.0, Storybook requires Svelte 4 and up.
 
+#### Preact
+
+##### Require Preact 10 and up
+
+Starting in 8.0, Storybook requires Preact 10 and up.
+
+##### No longer adds default babel plugins
+
+Until now, Storybook added a set of default babel plugins to the babel config for Preact projects which uses Webpack.
+
+We have configured the runtime automatic import plugin to automatically import `h` from `preact` when needed. This is no longer the case in Storybook 8.0. If you want to use the runtime automatic import plugin to skip unnecessary preact imports, you will need to add it to your babel config yourself. The same applies if you want to use TypeScript with Preact.
+
+```js
+.babelrc
+
+{
+  "plugins": [
+    [
+      // Add this to automatically import `h` from `preact` when needed
+      "@babel/plugin-transform-react-jsx", {
+        "importSource": "preact",
+        "runtime": "automatic" 
+      }
+    ],
+    // Add this if you want to use TypeScript with Preact
+    "@babel/preset-typescript"
+  ],
+}
+```
+
+If you want to configure the plugins only for Storybook, you can add the same setting to your `.storybook/main.js` file.
+
+```js
+const config = {
+  ...
+  babel: async (options) => {
+    options.plugins.push(
+      [
+        "@babel/plugin-transform-react-jsx", {
+          "importSource": "preact",
+          "runtime": "automatic" 
+        }
+      ], 
+      "@babel/preset-typescript"
+    )
+    return options;
+  },
+}
+
+export default config
+```
+
+We are doing this to apply the same babel config as you have defined in your project. This streamlines the experience of using Storybook with Preact. Additionally, we are not vendor-locked to a specific Babel version anymore, which means that you can upgrade Babel without breaking your Storybook.
+
+#### Web Components
+
+##### Dropping default babel plugins in Webpack5-based projects
+
+Starting in 8.0, Storybook doesn't apply any default babel plugins to Web Components projects. Storybook will pick up the babel configuration you have in place. Before Storybook 8.0, Storybook has added the following babel presets and plugins:
+
+- `@babel/preset-env`
+
+The following list of plugins are not necessary anymore, since they are included in `@babel/preset-env`:
+
+- `@babel/plugin-syntax-dynamic-import`
+- `@babel/plugin-syntax-import-meta`
```

Comment:
@valentinpalkovic, when you're able, can you adjust this to the following:
"
Until the 8.0 release, Storybook provided the `@babel/preset-env` preset for Web Component projects by default. This is no longer the case, as any Web Components project will use the configuration you've included. Additionally, if you're using either the `@babel/plugin-syntax-dynamic-import` or `@babel/plugin-syntax-import-meta` plugins, you no longer have to include them as they are now part of the `@babel/preset-env` preset.
"
Also, can you capitalize Babel in the heading and adjust the link accordingly?